docs/content/how-to/creating-catalogs.md

```diff
@@ -84,12 +84,15 @@ Paimon Hive catalog in Flink relies on Flink Hive connector bundled jar. You sho
 
 The following Flink SQL registers and uses a Paimon Hive catalog named `my_hive`. Metadata and table files are stored under `hdfs://path/to/warehouse`. In addition, metadata is also stored in Hive metastore.
 
+If your Hive requires security authentication such as Kerberos,LDAP,Ranger and so on. You can specify the hive-conf-dir parameter to the hive-site.xml file path
```

Comment:
`If your Hive requires security authentication such as Kerberos, LDAP, Ranger and so on. You can specify the hive-conf-dir parameter to the hive-site.xml file path.`